integration/resource_lacework_query_test.go

```diff
@@ -176,4 +231,22 @@ var (
         EVENT
     }        
 }`
+
+	queryDeprecatedSyntaxWithID = `Lql_Terraform_Query {
```

Comment:
I added one test to:
1. show that this method has been deprecated and
2. we still support it